README.md

```diff
@@ -68,6 +68,10 @@ implements `stream` and `exec`. The calling conventions and return parameters fo
 
 * `exec(cmd, args, opts)` should return `cmd`'s exit code. `exec` implementations should write their stdout/stderr chunks to stdout/stderr. Note that `STDIN`, if provided, can be accessed via the `opts[:stdin]` key.
 
+* `delete` should return `true` if the entry was deleted, `false` if the entry's deletion is in progress.
```

Comment:
Do we have any sort of testing in this repo that should be extended for delete and signal?

Reply:
Not really. `wash validate` is currently the way we test this repo, and that does not work w/ Signal and Delete.

Reply:
Could be worth exploring some spec tests, but they would also be working against a "spec" rather than actual code so not sure they'd help much.